deparse.c

```diff
@@ -2606,6 +2634,23 @@ sqlite_deparse_const(Const *node, deparse_expr_cxt *context, int showtype)
 				sqlite_deparse_string_literal(buf, extval);
 
 			break;
+		case UUIDOID: 
+			/* always deparse to BLOB because this is internal PostgreSQL storage 
+			 * the string for BYTEA always seems to be in the format "\\x##"
+			 * where # is a hex digit, Even if the value passed in is
+			 * 'hi'::bytea we will receive "\x6869". Making this assumption
+			 * allows us to quickly convert postgres escaped strings to sqlite
+			 * ones for comparison
+			 */
+			extval = OidOutputFunctionCall(typoutput, node->constvalue);
+			appendStringInfo(buf, "X\'");
+			for (int i = 0; i < strlen(extval); i++) {
```

Comment:
Could you fix this code style to:
```c
int i;

for (i = 0; i < strlen(extval); i++)
{
}
```
This code can make build error when using legacy gcc.

Reply:
Fixed in https://github.com/pgspider/sqlite_fdw/pull/82/commits/d4850bd4515a6fa380d73db2583ff04596ecf38c , @nxhai98 . Also some code blocks was added near of UUID deparsing. Please check again with legacy `gcc`.

Reply:
Thanks for your fixing. I confirmed.